test/plugins/validation/2and3/info.js

```diff
@@ -0,0 +1,98 @@
+const config = require('../../../../src/.defaultsForValidator').defaults.shared;
+const expect = require('expect');
+const {
+  validate
+} = require('../../../../src/plugins/validation/2and3/semantic-validators/info');
+
+describe('validation plugin - semantic - error handling for incorrect formatting', () => {
+  describe('there should be errors when the defintion is missing neccesrry fields', function() {
```

Comment:
"Describe" blocks are used to group related tests together. Since all of your tests are dealing with the `info` object in a pretty related way, you don't need to have this describe block - the top level one is enough